### PR TITLE
Updating official Kafra information from Rock Ridge

### PR DESCRIPTION
**Pull request by JohnnyPlayy**
* **Server Mode**: renewal

* **Description of Pull Request**: Information extracted from Iro, I noticed that this npc was incomplete.

### CODE REVIEW

**Comment by Atemo** [comment]
Anything left?

**Comment by JohnnyPlayy** [comment]
At least in Kafra it was all that was missing, already the missions, I did not get to do completely in the officer.
I could do what I had commented with you of separating what is not part of the mission and putting in place.

File: npc/re/kafras/kafras.txt [hunk 1]
```diff
@@ -110,7 +110,10 @@ job3_rune01,92,62,3	script	Kafra Employee::kaf_rune	114,{
 harboro1,291,207,5	script	Kafra Employee::kaf_rock	4_F_KAFRA1,{
 	cutin "kafra_01",2;
 	callfunc "F_KafSet";
-	callfunc "F_Kafra",0,8,2,500,0;
-	savepoint "harboro1",286,189,1,1;
-	callfunc "F_KafEnd",0,1,"in the city of Rockridge";
+	mes "[Kafra Employee]";
+	mes "Welcome to Kafra. We're always by your side.";
+	mes "How may I help you?";
```
Comment (Lemongrass3110) [inline npc/re/kafras/kafras.txt]: This looks like the default kafra message:
```
	default:// Default message (obsolete)
		mes "[Kafra Employee]";
		mes "Welcome to the";
		mes "Kafra Corporation.";
		mes "The Kafra services";
		mes "are always on your side.";
		mes "How may I assist you?";
		break;
```

Comment (JohnnyPlayy) [reply npc/re/kafras/kafras.txt]: After I wrote that I noticed

**Comment by Atemo** [comment]
Thanks !

**Pull request merged**